assembl/static2/js/app/components/administration/voteSession/tokenTypeForm.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import { BlockPicker } from 'react-color';
```

Comment:
maybe import as ColorPicker so it is very clear when we read the code that this is the color picker ?